index.html

```diff
@@ -213,7 +215,7 @@ <h2 class="heading">About</h2>
             <li>HTML</li>
             <li>CSS</li>
           </ul>
-          <img class="icon-bg" src="images/ic_languages_mob.svg" alt="#" />
+          <img class="icon-bg" src="images/ic_languages_mob.svg" alt="" />
         </li>
         <li class="red about heading3">
           Frameworks
```

Comment:
- [ ] In the mobile version of your page, the following section is not matched. Currently, your page has more space on the right side of this section. Please fix it.
**Optional:**
- [ ] Your design has no bullet for the items. In the Figma design, there are bullet points for every li. Please work on that.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![l1](https://user-images.githubusercontent.com/70483835/155174663-27b033ba-58d7-42dd-b70c-d5260d7f78ad.png) | ![l2](https://user-images.githubusercontent.com/70483835/155174726-92d2f17e-6768-428f-ab6a-c47b0f614146.png)